cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/wizard-basic/management-credential-step/ManagementCredentials.tsx

```diff
@@ -50,7 +48,7 @@ function ManagementCredentials(props: Partial<StepProps>) {
     const [regions, setRegions] = useState<AzureLocation[]>([]);
 
     const handleInputChange = (field: FormField, value: string) => {
-        if (field !== 'SSH_PUBLIC_KEY') {
+        if (field !== 'SSH_PUBLIC_KEY' && field !== 'REGION') {
```

Comment:
I think it'd be nice to use a field enumeration here instead of hard-coded 'REGION'